### DISCUSSION TITLE
Get count from Histogram using View

### DISCUSSION THREAD

**Discussion by KevinLiLu** [open-telemetry/opentelemetry-java#6528] [Q&A]
I am using metric views to collect and export metrics. I have a Histogram which is tracking request latencies, and I want to export the total request count as a separate metric.

The spec mentions this should be possible via customizing the aggregation for the view ([link](https://github.com/open-telemetry/opentelemetry-specification/blob/03e4ea2748e18e63d1e00e58e373ca55768fb1b0/specification/metrics/sdk.md?plain=1#L239-L243)):
> A View provides SDK users with the flexibility to customize the metrics that are output by the SDK. Here are some examples when a View might be needed:
...
Customize the aggregation - if the default aggregation associated with the Instrument does not meet the needs of the user. For example, an HTTP client library might expose HTTP client request duration as [Histogram](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#histogram) by default, but the application developer might only want the total count of outgoing requests.

How would this work in the Java SDK? I don't see a count aggregation in [Aggregation.java](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/Aggregation.java).

**Comment by jkwatson** [comment]
Unfortunately, there isn't a Count aggregation specified currently [https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md?plain=1#L558], and we don't currently support custom aggregations in the standard SDK. If this is something you need, I recommend opening up an issue in the specification to add a new aggregation type.
